modules/ROOT/pages/administration/user-management.adoc

```diff
@@ -0,0 +1,122 @@
+= NOM user management
+:description: This section describes the user management in Neo4j Ops Manager.
+
+*User Management* allows setting up NOM users for persons who are in charge of managing
+the Neo4j deployment and reflecting their roles by assigning appropriate access levels.
+
+These tasks can only be accomplished by a NOM user who has the role *NOM Admin*.
+
+[NOTE]
+====
+The default NOM user `admin` has the role *NOM Admin* and is therefore can be used to create new NOM users
+(including other *Administrators*) and configure their access to NOM features.
+====
+
+== Scope of NOM users
+
+NOM users can only be used to access the functionality of the NOM server and are not related to other systems
+(Neo4j database, operating system, etc.)
+
+== NOM user administration tasks
+
+The NOM user management is performed in the *Administrators* section of the NOM settings page.
+
+=== Creating a user
+
+Click on *"Create user"* button to open the sidebar containing the tabs *"User info"* and *"DBMS access control"*.
+
+==== Assigning access levels on features
+
+If a user has the role *NOM admin* (see checkbox on the *"User info"* tab), it will have full feature access, for
+all DBMSs. Therefore, the tab *"DBMS access control"* is not relevant for *NOM admins*.
+
+Other users will have the role *Administrator* assigned. In this case, access to every feature can be given explicitly or
+implicitly by assigning access levels to a DBMS glob pattern. Access levels for global features can be set on the
+*"User info"* tab and DBMS-specific features can be secured on the *"DBMS access control"*.
+See the following table for permission description.
+
+|===
+|NOM Feature (scope) |Access levels |Description
+
+|*Agent* (global)
+|No access, view, edit
+|Access to the *"Agents"* page of the NOM settings area.
+
+|*Status* (DBMS-specific)
+|View, edit
+|Access to the *"Status"* page for assigned DBMS(s).
+
+|*Security* (DBMS-specific)
+|No access, view, edit
+|Access to the *"Security panel"* page for assigned DBMS(s).
+
+|*Status* (DBMS-specific)
+|No access, view
+|Access to the *"Logs"* page for assigned DBMS(s).
+
+|*Upgrades* (DBMS-specific)
+|No access, View, edit
+|Access to the *"Upgrade"* page for assigned DBMS(s).
+|===
+
+[IMPORTANT]
+====
+If DBMS is renamed or migrated after being assigned explicit permissions, the permissions will remain unchanged.
+====
+
+==== Duplicating permissions of another user
+
+The function "Copy other's access" can be used in cases when several NOM users with similar permissions must be created.
+This function appends permissions of another user (selected in the popup) to the permissions of the current user. Permissions
+that where already assigned are not appended.
+
+==== Working with DBMS glob patterns
+
+Should the Neo4j deployment have many DBMSs, it can be efficient to assign DBMS-specific permissions to a set
```

Comment:
```suggestion
Should the Neo4j deployment have many DBMSs, it can be efficient to assign DBMS-specific permissions to a set of DBMSs with a single line in the permission table.
```